app/scripts/controllers/permissions/specifications.js

```diff
@@ -278,6 +279,16 @@ function validateCaveatAccounts(accounts, getInternalAccounts) {
   });
 }
 
+export const UnrestrictedEthSigningMethods = Object.freeze([
```

Comment:
Just to match the same naming convention of this file:

```suggestion
export const unrestrictedEthSigningMethods = Object.freeze([
```